dev-test/gql-tag-operations-masking-star-wars/gql/fragment-masking.ts

```diff
@@ -1,43 +1,48 @@
-import { TypedDocumentNode as DocumentNode } from '@graphql-typed-document-node/core';
+import { ResultOf, DocumentTypeDecoration } from '@graphql-typed-document-node/core';
```

Comment:
@beerose I think we can even delete this fixture as the new examples also cover this fixture test.

https://github.com/dotansimha/graphql-code-generator/blob/a109caea323ae778b60be4e0522caeb4e4de77be/.github/workflows/main.yml#L58